tests/cli/tests/simd_test.go

```diff
@@ -0,0 +1,128 @@
+package tests
```

Comment:
This file ends with `_test.go` yet the package's name is `tests`?
And how did we end up having a `tests` sub-directory within a `tests` package?

Reply:
@sahith-narahari l think we can just refactor tests/cli/helpers into tests/cli in the cli package and then put the tests in tests/cli under the cli_test package

Reply:
Yeah, this approach looks cleaner than the current one. Will make the changes accordingly